python/test/unit/hopper/test_experimental_tma.py

```diff
@@ -534,3 +537,93 @@ def alloc_fn(size: int, align: int, stream: Optional[int]):
     )
     torch.testing.assert_close(ref_out, A)
     assert "tensormap.cp_fenceproxy.global.shared::cta.tensormap::generic.release.gpu.sync.aligned" in kernel.asm["ptx"]
+
+
+@requires_tma
+def test_experimetal_descriptor_load_3d_no_jit():
+    """In addition to testing 3D TMA, we also test parsing of TTGIR when TMA descriptors are in arguments.
+    See https://github.com/triton-lang/triton/pull/4875
+    """
```

Comment:
I was not aware of https://github.com/triton-lang/triton/pull/4875 before I rebased my internal branch where I also updated the parser to support TMA desc input. I removed the parser change from this branch, but left this test case for completeness (since 4875 didn't add a test and I want to test 3D TMA too).

Reply:
A test was added by @SamGinzburg  in https://github.com/triton-lang/triton/pull/4924. Let's use that test for 3D instead of adding a new one?

See https://github.com/triton-lang/triton/blob/main/python/test/unit/tools/test_irsource.py